### PR TITLE
Create the placeholder for the Recommendations endpoint

### LINKED ISSUE #19: Add Placeholder for Recommendation Index Page
- [ ] There is a page at the endpoint recommendations/list.
- [ ] There is a page at the endpoints recommendations/create and recommendations/edit/:id
- [ ] The endpoints at recommendations/create and recommendations/edit can only be accessed by an admin user.
- [ ] The changes passes npm test and mutation coverage testing.

### PR DESCRIPTION
**Pull request by minhbui27**
In this commit, I modified AppNavbar.js to add routes for the endpoints at recommendations/create, recommendations/index and recommendations/edit/:id.
Closes #19

### CODE REVIEW

**Review by wesleykam — DISMISSED** [review]
lgtm assuming all tests pass after merge conflicts resolved

**Review by KaieChen — APPROVED** [review]
lgtm!

**Pull request merged**
